zuul-core/src/main/java/com/netflix/zuul/message/http/Cookies.java

```diff
@@ -16,12 +16,19 @@
 
 package com.netflix.zuul.message.http;
 
+import com.netflix.config.CachedDynamicBooleanProperty;
+import com.netflix.config.DynamicStringProperty;
+import com.netflix.zuul.message.Headers;
 import io.netty.handler.codec.http.Cookie;
+import io.netty.handler.codec.http.DefaultCookie;
+import io.netty.handler.codec.http.cookie.ServerCookieDecoder;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
The style guide we use suggests not using wildcar imports.

Reply:
Addressed.